### PR TITLE
remove solpp dependencies

### PR DESCRIPTION
**Pull request by vladenysiuk**
# What ❔

Removing deprecated solpp dependencies from l2-contracts

### CODE REVIEW

**Review by benceharomi — COMMENTED** [review]
I see that you removed `hardhat-solpp` from the `package.json`, please update `yarn.lock`

File: l2-contracts/hardhat.config.ts [hunk 1]
```diff
@@ -37,6 +45,15 @@ export default {
   solidity: {
     version: "0.8.20",
   },
+  solpp: {
```
Comment (benceharomi) [inline l2-contracts/hardhat.config.ts:48]: why is this needed?

Comment (vladenysiuk) [reply l2-contracts/hardhat.config.ts]: Thought, that it passes some params to deployer. Now it's removed.

**Review by StanislavBreadless — APPROVED** [review]
